pkg/rc/replication_controller.go

```diff
@@ -410,12 +404,23 @@ func (rc *replicationController) alertInfo(rcFields fields.RC, msg string) alert
 }
 
 func (rc *replicationController) removePods(rcFields fields.RC, current types.PodLocations, eligible []types.NodeName) error {
+	// If the RC is disabled, we don't want to remove any pods with one
+	// exception: if the ReplicasDesired count is zero and their are "ineligible"
+	// nodes. In this case, we want to unschedule all ineligible nodes.
+	//
+	// An ineligible node is defined as a node that appears in "current" but not
+	// in "eligible". This means that the scheduler has decided this node is no
+	// longer eligible.
+	if rcFields.ReplicasDesired != 0 && rcFields.Disabled {
```

Comment:
hmm actually i think we want to just rely on the transfer finishing instead of having this now